python/private/pypi/extension.bzl

```diff
@@ -525,7 +526,72 @@ supported from this version without extra handling from the user.
 :::
 """,
     ),
-} | AUTH_ATTRS
+} | AUTH_ATTRS | {
+    # Downloader options
+    "downloader": attr.string(
```

Comment:
I would say `bazel_downloader` is better. As is, it kinda sounds like it would modify downloading altogether (similar to `pip.parse.download_only`).